release/.bumpversion-rc.cfg

```diff
@@ -1,17 +1,13 @@
 [bumpversion]
 parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(\-rc\.(?P<rc>\d+))?
-serialize = 
+serialize =
 	{major}.{minor}.{patch}-rc.{rc}
 	{major}.{minor}.{patch}
 current_version = 4.8.0
 commit = true
 tag = true
 
-[bumpversion:file:../components/frontend/package.json]
-search = "version": "{current_version}",
-replace = "version": "{new_version}",
-
-[bumpversion:file:../components/renderer/package.json]
+[bumpversion:glob:../components/*/package.json]
```

Comment:
Perhaps also use `../**/package.json` here, e.g. the docs also have a `package.json`

Reply:
Yes, but those don't contain the Quality-time version number.